src/main/java/uk/gov/hmcts/cmc/claimstore/services/staff/ClaimIssuedStaffNotificationService.java

```diff
@@ -95,35 +94,29 @@ private EmailData prepareEmailData(Claim claim, Optional<String> defendantPin, S
         return emailAttachments;
     }
 
-    private EmailAttachment sealedLegalClaimPdf(Claim claim) {
-        byte[] generatedPdf = pdfServiceClient.generateFromHtml(
-            staffEmailProperties.getEmailTemplates().getLegalSealedClaim(),
-            legalSealedClaimContentProvider.createContent(claim)
-        );
+    private EmailAttachment sealedLegalClaimPdf(final Claim claim) {
+        final byte[] generatedPdf = sealedClaimDocumentService.generateLegalSealedClaim(
+            claim.getExternalId());
 
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused method parameter "authorisation". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.reform.hmcts.net/coding_rules#rule_key=squid%3AS1172)

Reply:
done